recipes/nbrefactor/recipe.yaml

```diff
@@ -0,0 +1,56 @@
+schema_version: 1
+
+context:
+  name: nbrefactor
+  version: 0.1.6
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/nbrefactor-${{ version }}.tar.gz
+  sha256: cf09d18919a444a4c7430e8a37fbe0b3ea8fbb745e6471d3a4e1e11f11f396d5
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  python:
+    entry_points:
+      - jupyter-nbrefactor = nbrefactor.cli:main
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - setuptools >=42
+    - wheel
+    - pip
+  run:
+    - python >=${{ python_min }}
+    - python-graphviz
+    - nbformat
+    - tqdm
+
+tests:
+  - python:
+      imports:
+        - nbrefactor
+      pip_check: true
+      python_version: ${{ python_min }}.*
+  - requirements:
+      run:
+        - python ${{ python_min }}.*
```

Comment:
I believe that the line above covers this.

```suggestion
```

Reply:
Added as instructed by @synapticarbors but I can pull it back out 🫡